doc/classes/Navigation.xml

```diff
@@ -55,6 +61,7 @@
 			<argument index="2" name="optimize" type="bool" default="true">
 			</argument>
 			<description>
+				Returns a path of points as a [code]PoolVector3Array[/code]. If the third optional [code]bool[/code] parameter is passed as false the [code]NavigationMesh[/code] agent properties will be taken into account, otherwise it will return the nearest path and ignore agent radius, height, etc.
```

Comment:
> If the third optional `bool` parameter is passed as false

IMO that's too long. Maybe: If `optimize` is false...

Reply:
Thanks, I agree :)